### ISSUE TITLE
Will it automatically update browser when new version of a core is released by BAS?

### ISSUE THREAD

**Issue by TheGP** [CheshireCaat/puppeteer-with-fingerprints#110]

**Comment by CheshireCaat** [comment]
Yes, the engine is updated automatically, with rare exceptions - so I still recommend updating the plugin periodically.

**Comment by TheGP** [comment]
@CheshireCaat update how? by removing data dir?

**Comment by CheshireCaat** [comment]
@TheGP The engine is always updated automatically at the next launch, but there may be a case when certain versions of the plugin (past versions) will not have updates for it.

You don't need to do anything manually in most cases, just try to update the library itself more often, just in case.
